### DISCUSSION TITLE
Compatibility Matrix

### DISCUSSION THREAD

**Discussion by vikashg** [Project-MONAI/monai-deploy-app-sdk#363] [General]
Hello all, 
Do you think we should have a compatibility matrix for different components of monai 

https://projectcontour.io/resources/compatibility-matrix/
@MMelQin, @wyli, @aihsani, @dbericat, @mocsharp. Thoughts?

Thanks 
Vikash

**Comment by MMelQin** [comment]
This may sound like an excuse, but typically before the software V1.0 is released, APIs and dependencies can change and change rapidly, and backwards compatibility may not be supported or a clear and verified compatibility matrix is published. The App SDK is yet to reach V1.0 milestone.

The App SDK is largely based on Python packages, and the requirements are documented for the minimal, development, as we as running examples. The MONAI App Package, however, depends on NVIDIA Deep Learning container images, specifically the [PyTorch Python container images](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/pytorch),  which themselves are covered in this published [Framework Containers Support Matrix](https://docs.nvidia.com/deeplearning/frameworks/support-matrix/index.html)

The App SDK shall strive to be compatible with the latest and recently released Deep Learning container images, while also maintaining compatibility with MONAI Core, which is the top most dependency of App SDK AI related functionalities.
